File: picture/picture/Form1.vb
Public Class Form1

    Private Sub Button1_Click(sender As System.Object, e As System.EventArgs) Handles Button1.Click
        PictureBox1.Size = New Size(500, 500)
    End Sub

    Private Sub Button2_Click(sender As System.Object, e As System.EventArgs) Handles Button2.Click
        Dim x, y As Integer
        x = PictureBox1.Width + 5
        y = PictureBox1.Height + 5
        PictureBox1.Size = New Size(x, y)
    End Sub

    Private Sub Button3_Click(sender As System.Object, e As System.EventArgs) Handles Button3.Click
        Dim x, y As Integer
        x = PictureBox1.Width - 5
        y = PictureBox1.Height - 5
        PictureBox1.Size = New Size(x, y)
    End Sub

    Private Sub Button4_Click(sender As System.Object, e As System.EventArgs) Handles Button4.Click
        Dim x, y As Integer
        Dim answer As String
        answer = InputBox("Which direction do you want the picturebox to move?                                                                   your options are: left, right, up and down")
        answer = answer.ToLower()
        If answer = "up" Then
            x = PictureBox1.Location.X + 0
            y = PictureBox1.Location.Y - 10
        ElseIf answer = "down" Then
            x = PictureBox1.Location.X + 0
            y = PictureBox1.Location.Y + 10
        ElseIf answer = "right" Then
            x = PictureBox1.Location.X + 10
            y = PictureBox1.Location.Y + 0
        ElseIf answer = "left" Then
            x = PictureBox1.Location.X - 10
            y = PictureBox1.Location.Y + 0
        Else
            MessageBox.Show("I told you the options, why didn't you choose one of those options?")
        End If
        PictureBox1.Location = New Point(x, y)
    End Sub

    Private Sub Button5_Click(sender As System.Object, e As System.EventArgs) Handles Button5.Click
        Dim x, y As Integer
        PictureBox1.Size = New Size(588, 392)
        x = PictureBox1.Location.X
        y = PictureBox1.Location.Y
        PictureBox1.Location = New Point(12, 12)
    End Sub
End Class

File: WindowsFormsApplication3/WindowsFormsApplication3/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication3
{
    public partial class Form1 : Form
    {
        int x = 50 ;
        int y = 165;
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string answer = textBox1.Text;
            answer = textBox1.Text.ToLower();
            if (answer == "chair")
            {
                MessageBox.Show("You are correct");
            }
            else
            {
                MessageBox.Show("I don't know about that one");
            }
        }

        private void button2_Click(object sender, EventArgs e)
        {
            if (radioButton3.Checked == true)
            {
                MessageBox.Show("You are correct");
            }//end if
            else
            {
                MessageBox.Show("That is scary");
            }//end else

        }

        private void button3_Click(object sender, EventArgs e)
        {
            pictureBox1.Size = new Size(300, 300);
            pictureBox1.Location = new Point(0, 0);
        }

        private void button4_Click(object sender, EventArgs e)
        {
            if (checkBox1.Checked & checkBox2.Checked == true & checkBox3.Checked == false)
            {
                MessageBox.Show("that is right");
            }
            else if (checkBox1.Checked & checkBox2.Checked == true & checkBox3.Checked == true)
            {
                MessageBox.Show("wrong");
            }
            else if (checkBox1.Checked & checkBox3.Checked == true)
            {
                MessageBox.Show("Wrong");
            }
            else if (checkBox2.Checked & checkBox3.Checked == true)
            {
                MessageBox.Show("Wrong"); 
            }
            else if (checkBox1.Checked == true)
            {
                MessageBox.Show("I said choose 2");
            }
            else if (checkBox2.Checked == true)
            {
                MessageBox.Show("I said choose 2");
            }
            else if (checkBox3.Checked == true)
            {
                MessageBox.Show("I said choose 2");
            }
        }

        private void button5_Click(object sender, EventArgs e)
        {
            if (radioButton4.Checked == true)
                {
                y--;
                pictureBox1.Location = new Point(50,y);
                }
            if (radioButton5.Checked == true)
            {
                y++;
                pictureBox1.Location = new Point(50, y);
            }
            if (radioButton6.Checked == true)
            {
                x--;
                pictureBox1.Location = new Point(x, 165);
            }
            if (radioButton7.Checked == true)
            {
                x++;
                pictureBox1.Location = new Point(x, 165); 
            }
        }
    }
}
